core/dbt/task/init.py

```diff
@@ -20,11 +28,7 @@
 IGNORE_FILES = ["__init__.py", "__pycache__"]
 
 ON_COMPLETE_MESSAGE = """
-Your new dbt project "{project_name}" was created! If this is your first time
-using dbt, you'll need to set up your profiles.yml file -- this file will tell dbt how
-to connect to your database. You can find this file by running:
-
-  {open_cmd} {profiles_path}
+Your new dbt project "{project_name}" was created!
```

Comment:
(low priority)

This message feels like _lot_ of text after the interactive bite-sized chunks. I wonder if we can do something cool with `click`, to either:
- [clear](https://click.palletsprojects.com/en/8.0.x/api/#click.clear) the terminal, leaving only the welcome message
- prompting with each item one by one, so that the user "acknowledges" each: project created!, here's a link to the docs, need help?, happy modeling!